cardano-cli/test/cardano-cli-golden/files/golden/help/latest_governance_drep_registration-certificate.cli

```diff
@@ -25,5 +26,9 @@ Available options:
   --drep-metadata-url TEXT DRep anchor URL
   --drep-metadata-hash HASH
                            DRep anchor data hash.
+  --check-drep-metadata-hash
+                           Check the DRep metadata hash (from
```

Comment:
"A check that calculates a hash and compares it to the specified hash" ? This wording could probably be improved but what does "Check" mean? We need to be clearer.

Reply:
Maybe the confusing bit is that it seems like a noun "check" while I meant the imperative of the verb "to check". I will write "verify that", that should be clearer